fusion_power_video.cc

```diff
@@ -106,10 +108,10 @@ chunk flags meanings:
  for all frames.
 
 image flags meanings:
--flags & 1: iff true, delta frame prediction is enabled. This must be false if
+-flags & 1: if true, delta frame prediction is enabled. This must be false if
```

Comment:
iff was not a typo but "if and only if". Nothing against the change though.

Reply:
Oh... I did not know that idiom... Learning something new each day - thanks!